src/main/java/domain/LottoNumber.java

```diff
@@ -0,0 +1,66 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+	private static final int MIN_LOTTO_NUMBER = 1;
+	private static final int MAX_LOTTO_NUMBER = 45;
+	private static final List<LottoNumber> CACHE = new ArrayList<>();
+
+	private final int value;
+
+	static {
+		for (int i = MIN_LOTTO_NUMBER; i <= MAX_LOTTO_NUMBER; i++) {
+			CACHE.add(new LottoNumber(i));
+		}
+	}
```

Comment:
LottoNumber를 미리 캐싱해둔거니 `LOTTO_NUMBERS`라고 네이밍해보시는건 어떨까요? 🙌

추가적으로 아래와 같이 CACHE를 비어있는 ArrayList로 미리 선언해두는 것이 아니라, 
캐싱해두려는 List<LottoNumber>를 생성한 뒤, 할당해주는 방식으로 변경해볼 수 있을 것 같아요. 🙂

```suggestion
	private static final List<LottoNumber> CACHE;

	private final int value;

	static {
            CACHE = IntStream.rangeClosed(MIN_LOTTO_NUMBER, MAX_LOTTO_NUMBER)
                .mapToObj(LottoNumber::new)
                .collect(Collectors.toList());
        }
```